gen/calc.py

```diff
@@ -849,12 +849,12 @@ def calculate_check_config(check_time):
                 'ip_detect_script': {
                     'description': 'The IP detect script produces valid output',
                     'cmd': ['/opt/mesosphere/bin/dcos-checks', 'ip'],
-                    'timeout': instant_check_timeout
+                    'timeout': normal_check_timeout
                 },
                 'docker': {
                     'description': 'Docker is installed',
                     'cmd': ['/opt/mesosphere/bin/dcos-checks', 'executable', 'docker'],
-                    'timeout': '1s'
+                    'timeout': normal_check_timeout
```

Comment:
Tim requested in the JIRA ticket that this be changed. This check is not in enterprise or in 1.12, 1.11. So it's only changed here.